taxi/views.py

```diff
@@ -67,6 +86,17 @@ class CarCreateView(LoginRequiredMixin, generic.CreateView):
     fields = "__all__"
     success_url = reverse_lazy("taxi:car-list")
 
+    def get_form(self, form_class=None):
+        if not form_class:
+            form_class = self.get_form_class()
+        form = super().get_form(form_class)
+        form.fields["drivers"] = forms.ModelMultipleChoiceField(
+            queryset=Driver.objects.all(),
+            widget=forms.CheckboxSelectMultiple,
+            required=True,
+        )
+        return form
+
```

Comment:
For better separation of concerns and adhering to the principles of modularity and reusability, modifying the form directly rather than within the view is a preferable approach.
I meant simply add this field inside form, not inside view, that seems more logical